### ISSUE TITLE
Acknowledgement and suggestion

### ISSUE THREAD

**Issue by tisonkun** [Microflash/remark-callout-directives#1]
Hi @naiyerasif!

Thank you so much that this lib satisfies my requirements perfectly!

I'm building a blog with Astro, MDX and now including this lib. Once the blog is online, I'll share a link here.

Two suggestions here:

1. The document lacks a bit about how to apply themes. I spend a few time to understand that the `index.js` is the options that I should pass to the plugin function in the way how the framework does it. And the CSS should be imported or copied to the final CSS file.
2. All the three preset themes absolutely meet their style. However, I may be more familiar with [Docusaurus' Admonitions](https://docusaurus.io/docs/markdown-features/admonitions). I'd appreciate it if you can think of bundling this theme, but it's somewhat a request so you make the decision :D

**Comment by tisonkun** [comment]
> Once the blog is online, I'll share a link here.

Here is my repo - https://github.com/tisonkun/dacapo/blob/e224b4c4fd45b0261a3ff9602f15c55417c0d051/astro.config.mjs#L19-L33

**Comment by naiyerasif** [comment]
> 1. The document lacks a bit about how to apply themes. I spend a few time to understand that the index.js is the options that I should pass to the plugin function in the way how the framework does it. And the CSS should be imported or copied to the final CSS file.

Yes, I admit that documentation might be a bit lacking. I'm planning to create a new release with some additional information on how to use themes. If you'd like to improve something specific, please create a PR with your suggested improvements.

> 2. All the three preset themes absolutely meet their style. However, I may be more familiar with [Docusaurus' Admonitions](https://docusaurus.io/docs/markdown-features/admonitions).

I removed Docusaurus' Infima theme with v4 of this plugin. I wanted to ship just enough themes to demonstrate the possibilities with this plugin, and let people remix their own themes.

> I'd appreciate it if you can think of bundling this theme, but it's somewhat a request so you make the decision :D

I have created a dedicated repository where you can play with Infima theme using GitHub Codespaces. Try it out and see if this works for you.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/naiyerasif/remark-callout-directives-themes)
